main.cpp

```diff
@@ -28,7 +28,19 @@ int main(){
       cout << "=" << num1*num2 << endl;
     }
     else if( input == "divide" ){
-      cout << "=" << num1/num2 << endl;
+      int remainder = num1%num2;
```

Comment:
Modulo operation should be done after num2 is checked to != 0, will cause floating point error.